### PR TITLE
Upgrade 3.x packages

### PR DESCRIPTION
**Pull request by gauravtiwari**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,38 +12,38 @@
     "yarn": ">=0.25.2"
   },
   "dependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
```
Comment (swrobel) [inline package.json]: Why the hard version dependencies?

**Comment by swrobel** [comment]
For some reason this is causing serious problems for me with yarn 1.6

https://github.com/yarnpkg/yarn/issues/5749#issuecomment-386335952

**Comment by gauravtiwari** [comment]
@swrobel Just released 3.5.3 to fix this ^^

File: package.json (hunk 1 above)
Comment (gauravtiwari) [reply package.json]: Sorry about this probably sneaked in while I was trying to update some packages between different branches.

**Pull request merged**